awx/ui_next/babel.config.js

```diff
@@ -1,18 +0,0 @@
-module.exports = api => {
-  api.cache(false);
-  return {
-    plugins: [
```

Comment:
I guess we are now relying on whatever CRA deems as the babel stuff we can do.  I'm alright with that, provided they keep up with the types of new language features we want.  But do any of these not work anymore, and we need to update code to not include them?

Reply:
I think removing this file _did_  cause an issue (see https://github.com/ansible/awx/pull/6954#issuecomment-627017618) but with the latest commit I pushed it, it should be find.

>  I'm alright with that, provided they keep up with the types of new language features we want.

Yep, exactly. Let's use it for now and if there ends up being some feature we absolutely have to add then we can weigh the pros and cons of not having the feature vs `ejecting` the configuration.

Reply:
makes sense to me.  my guess is it won't be a problem, they'll probably be ahead of the curve on that stuff

Reply:
ah actually, it looks like the babel macros (like the styled components import to get css to work) were created for the express purpose of getting new language features without having to eject from CRA/other zero-config build systems.  So if this situation pops up, we can just look if there is a macro for whatever we want to add.

https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros